coretex/cli/modules/utils.py

```diff
@@ -17,20 +17,45 @@
 
 from typing import List, Any, Optional, Callable
 from functools import wraps
+from importlib.metadata import version as getLibraryVersion
 
 import sys
 
 from py3nvml import py3nvml
 
 import click
+import requests
 
+from . import ui
 from ...utils.process import command
 
 
 def updateLib() -> None:
     command([sys.executable, "-m", "pip", "install", "--no-cache-dir", "--upgrade", "coretex"], ignoreStdout = True, ignoreStderr = True)
 
 
+def fetchLatestVersion() -> Optional[str]:
+    url = f"https://pypi.org/pypi/coretex/json"
+    response = requests.get(url)
+    if response.status_code == 200:
+        data = response.json()
+        return str(data['info']['version'])
+    else:
+        return None
```

Comment:
No need for else. You can use `response.ok` instead of checking `response.status_code` directly